From 00773e89893e4aaed721269eee0dbf844edd291c Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sat, 29 Sep 2018 12:56:14 +0200
Subject: [PATCH] fix build with glibc 2.28

Include to sys/sysmacros.h is needed to use makedev since glibc 2.28 so
add this include to devname.c and raidscan.c

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 libblkid/devname.c | 1 +
 raidscan.c         | 1 +
 2 files changed, 2 insertions(+)

diff --git a/libblkid/devname.c b/libblkid/devname.c
index d11bddd..12b63db 100644
--- a/libblkid/devname.c
+++ b/libblkid/devname.c
@@ -37,6 +37,7 @@
 #if HAVE_SYS_MKDEV_H
 #include <sys/mkdev.h>
 #endif
+#include <sys/sysmacros.h>
 #include <time.h>
 
 #include "blkidP.h"
diff --git a/raidscan.c b/raidscan.c
index fab3b02..6e07b3d 100644
--- a/raidscan.c
+++ b/raidscan.c
@@ -22,6 +22,7 @@
 #include "config.h"
 
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <linux/major.h>
-- 
2.17.1

